controllers/absence_prometheusrule.go

```diff
@@ -273,37 +271,24 @@ func (r *PrometheusRuleReconciler) updateAbsenceAlertRules(ctx context.Context,
 
 	unmodifiedAbsencePromRule := absencePromRule.DeepCopy()
 
-	// Step 3: get defaults for support group, tier and service labels and add them to the
-	// AbsencePrometheusRule.
-	//
-	// We make a copy of the existing CCloud labels so that we can compare if the labels
-	// have been updated.
-	labelOpts := LabelOpts{Keep: r.KeepLabel}
-	if keepCCloudLabels(labelOpts.Keep) {
-		var err error
-		labelOpts, err = r.labelOptsWithCCloudDefaults(ctx, promRule)
-		if err != nil {
-			return err
-		}
-
-		// Update the labels on AbsencePrometheusRule object in case they might've changed
-		// or delete them in case they no longer exist and defaults could not be
-		// determined.
-		// New CCloud format:
-		updateLabel(absencePromRule.Labels, LabelCCloudSupportGroup, labelOpts.DefaultSupportGroup)
-		updateLabel(absencePromRule.Labels, LabelCCloudService, labelOpts.DefaultService)
-		// Old CCloud format:
-		updateLabel(absencePromRule.Labels, LabelTier, labelOpts.DefaultTier)
-		updateLabel(absencePromRule.Labels, LabelService, labelOpts.DefaultService)
+	// Remove resource level tier, service, and support-group labels from existing PrometheusRule
+	// objects created by the operator.
+	// TODO: remove this in the next release when the labels have been removed from all
```

Comment:
We probably need to carry this a bit longer, to not accidentally skip this if one of the deployments doesn't receive the next update but only the one after that.

Reply:
I've updated the TODO comment.